downstairs/src/region.rs

```diff
@@ -1735,67 +1932,14 @@ pub(crate) mod test {
         assert!(!validate_repair_files(2, &good_files));
     }
 
-    #[test]
-    fn validate_repair_files_duplicate_pair() {
-        // duplicate file names for extent 2
-        let good_files: Vec<String> = vec![
-            "002".to_string(),
-            "002".to_string(),
-            "002.db".to_string(),
-            "002.db".to_string(),
-        ];
-        assert!(!validate_repair_files(2, &good_files));
-    }
-
-    #[test]
-    fn validate_repair_files_quad_duplicate() {
-        // This is an expected list of files for an extent
-        let good_files: Vec<String> = vec![
-            "001".to_string(),
-            "001.db".to_string(),
-            "001.db-shm".to_string(),
-            "001.db-shm".to_string(),
-        ];
-        assert!(!validate_repair_files(1, &good_files));
-    }
-
     #[test]
     fn validate_repair_files_offbyon() {
         // Incorrect file names for extent 2
-        let good_files: Vec<String> = vec![
-            "001".to_string(),
-            "001.db".to_string(),
-            "001.db-shm".to_string(),
-            "001.db-wal".to_string(),
-        ];
+        let good_files: Vec<String> = vec!["001".to_string()];
 
         assert!(!validate_repair_files(2, &good_files));
     }
 
-    #[test]
-    fn validate_repair_files_too_good() {
-        // Duplicate file in list
```

Comment:
This test did catch a case where I sent the same name twice.  You don't think we should have something
similar here?

Reply:
Now that we're down to a single file, that condition is checked in `validate_repair_files_duplicate`.  I also just added `validate_repair_files_old`, which ensures that we fail if the repair file list includes a `.db` file.